site/en/guides/border-router/external-commissioning/cli.md

```diff
@@ -72,7 +117,49 @@ device.
 
 ## Join the Thread network
 
-Next, on the Joiner device, [join the Thread network](join.md) and test network connectivity.
+Next, on the Joiner device, [join the Thread network](join.md) and test network
+connectivity.
+
+## Troubleshooting
+
+If you're having issues with OT Commissioner, make sure you use `stop`, then
+`exit` to properly end a session:
+
+```
+> stop
+[done]
+> exit
+```
+
+**IO_ERROR: NET - Reading information from the socket failed**
+
+The socket APIs return this error message when a call to bind or connect to OTBR
+fails. If you're receiving this error message, try the following:
+
+*   Make sure that you're passing the correct port number when you start OT
+    Commissioner. OTBR may use a different port after it's restarted or you
+    reboot your platform.
+*   Make sure that OTBR is running and that your Thread network is properly
+    configured, including your PSKc. Your Passphrase/Commissioner Credential
+    must be a string between 6 and 255 characters.
+*   Check your global IP addresses, for example `ifconfig eth0`. You may be
+    using the wrong IP address to start OT Commissioner.
+
+**SECURITY: SSL - A fatal alert message was received from our peer**
+
+OT Commissioner establishes a secure DTLS session with the border agent service.
+A fatal SSL error typically indicates that the secure DTLS session fails. If you
+receive this message, try the following:
+
+*   Check the `commissioner.log`, if available.
+*   Make sure that OT Commissioner is properly configured when you pass your
+    Non-CCM or CCM Thread network JSON file.
+*   If OT Commissioner wasn't shut down properly or you disconnected from an SSH
+    session, you may need to restart the app or your platform.
```

Comment:
The same, this should be a bug in OT Commissioner :)

Reply:
OK - I think the issue is when you lose network connectivity, then get it back. I'll reproduce & document my steps...